huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,109 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md 
+# refer to the above page to pull latest Pytorch image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu113-ubuntu20.04-sagemaker
```

Comment:
```suggestion
FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.11-gpu-py38-cu113-ubuntu20.04-sagemaker
```

Reply:
So we can reuse this Dockerfile for minor releases with no changes.

Reply:
Why do we prefer 1.11.0 (fully qualified version) over 1.11 (short version)?

Reply:
@saimidu @tejaschumbalkar Can we please get some clarity on this ? What should be the container tag for the short version 1.11 ?

Reply:
you can check all tag in the release note here, some of them is using short version
https://github.com/aws/deep-learning-containers/releases/tag/v1.1-pt-sagemaker-1.11.0-tr-py38